step_two_utils.py

```diff
@@ -10,6 +10,66 @@
 log_dir = os.environ.get('PY_GOOGLE_AUTH_LOG_PATH')
 
 
+def handle_prompt_error(response):
+    '''
+    This function checks for errors (if any) while using google prompt method for login.
+    '''
+    if "Sign-in canceled" in response.text:
+        error = 412
+
+    else:
+        file_name = utils.log_error("google prompt", response.text)
+        error = 500
+
+    return error
+
+
+def handle_otp_error(response, session):
+    '''
+    This function checks for errors (if any) while using google authenticator method for login.
+    '''
+
+    # TODO: shift these to config file
```

Comment:
yeah, will fix in another issue as there are many such things to handle through config.